### DISCUSSION TITLE
CVPR Tutorial - Demos results (Use just the 19th of June)

### DISCUSSION THREAD

**Discussion by paularamo** [openvinotoolkit/openvino_notebooks#569] [Show and tell]
We want to create a better dynamic with the audience, we will use this thread for sharing results during the tutorial.

**Comment by hcarrillo** [comment]
Apple M1 Pro 15.4 FPS <img width="1135" alt="Screen Shot 2022-06-19 at 1 58 22 PM" src="https://user-images.githubusercontent.com/2282970/174496377-ba3c0ccf-16b7-42a6-9456-40b1367e82d9.png">

**Reply by raymondlo84** [reply]
Try FP16! :) Take out the INT8 ;) It can go faster!

**Comment by rvainshtein** [comment]
Quick & dirty way to insert inference on a picture of a cat 😉
![image](https://user-images.githubusercontent.com/39670084/174496740-45d642bb-ea0a-4055-8f5c-5d3fcfe03d02.png)

**Reply by raymondlo84** [reply]
:D !!!!

**Comment by hcarrillo** [comment]
Task # 2
<img width="1135" alt="Screen Shot 2022-06-19 at 2 10 50 PM" src="https://user-images.githubusercontent.com/2282970/174496811-0def4ff4-9a07-4f32-aa7b-766a195a60e6.png">

**Comment by manuelf23** [comment]
Macbook air with M1
<img width="1422" alt="image" src="https://user-images.githubusercontent.com/32891101/174498306-03c8dab1-2492-44f5-99ae-540666b69cdf.png">

**Reply by raymondlo84** [reply]
Try FP16 without INT8! :) It can actually go a bit faster....

**Comment by manuelf23** [comment]
Mac book air M1
<img width="1398" alt="image" src="https://user-images.githubusercontent.com/32891101/174500596-286f08a7-b5ea-4ecc-a265-23c582a9b461.png">

**Reply by lmzh123** [reply]
Macbook Pro M1 running on Docker, no GPU detected
<img width="729" alt="image" src="https://user-images.githubusercontent.com/10107712/174500756-aa18b73e-a25b-4bab-ba46-bf7e3edcb238.png">

**Reply by raymondlo84** [reply]
Oh wow... Docker.....

**Reply by ryanloney** [reply]
OpenVINO does not support Apple M1 GPU. An Intel integrated GPU is required.

**Comment by paularamo** [comment]
Thanks All for your participation in our tutorial!
